docs/antora.yml

```diff
@@ -0,0 +1,20 @@
+name: maven-plugin
+title: Asciidoctor Maven Plugin
+version: '2.1'
+asciidoc:
+  attributes:
+    release-version: 2.1.0
+    maven-site-plugin-version: 3.9.0
+    project-repo: asciidoctor/asciidoctor-maven-plugin
+    uri-repo: https://github.com/{project-repo}
```

Comment:
You can't use attribute references in a component descriptor.

Reply:
Luckily I see `uri-repo` is only required for the README, so we can remove it. Thanks!